### DISCUSSION TITLE
How do I use expressivity to get audio ?

### DISCUSSION THREAD

**Discussion by helloworld53** [facebookresearch/seamless_communication#272] [Q&A] [answered]
I have pip installed the setup as per this repo. I also have access to the seamless expressive's model and its checkpoints. How should I merge these into the main file To do a expressivity inference

**Comment by kauterry** [comment, marked as answer]
Check out this section: https://github.com/facebookresearch/seamless_communication/tree/main?tab=readme-ov-file#seamlessexpressive-inference

Check out the SeamlessExpressive section in the tutorial as well: https://github.com/facebookresearch/seamless_communication/blob/main/Seamless_Tutorial.ipynb
